README.rst

```diff
@@ -48,6 +48,12 @@ In your ``conf.py`` file:
 
     html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 
+or (since master):
```

Comment:
I think this text should be handled differently maybe use `.. versionchanged::` or `.. versionadded::` referencing the next major release of the theme (0.2.5)